.travis.yml

```diff
@@ -144,6 +145,10 @@ install:
     - git clone git://github.com/astropy/ci-helpers.git
     - source ci-helpers/travis/setup_conda.sh
 
+    # Use this until https://github.com/sphinx-gallery/sphinx-gallery/pull/255
+    # is merged.
+    - if [[ $SETUP_CMD == *build_docs* ]]; then pip -q install git+https://github.com/pllim/sphinx-gallery.git@stat-iter-deprec#egg=sphinx-gallery; fi
```

Comment:
Actually I am not sure. The test log reported failure and the total number of warnings but did not say *which* ones counted. Should I undo this and see what happens?